hasjob/views/helper.py

```diff
@@ -489,11 +548,9 @@ def list_dirty_impression_counts():
 
 def update_impression_counts(jobpost_ids):
     for jobpost_id in jobpost_ids:
-        # Replicate post.viewcounts's query here because we don't want to load the post from db just to
-        # access a class instance method
-        redis_store.hset('hasjob/viewcounts/%d' % jobpost_id, 'impressions',
-            db.session.query(db.func.count(db.func.distinct(EventSession.user_id)).label('count')
-            ).join(JobImpression).filter(JobImpression.jobpost_id == jobpost_id).first().count)
+        redis_store.hset('hasjob/viewcounts/%d' % jobpost_id,
```

Comment:
You're replicating the key syntax here but (a) not mentioning the existing `JobPost.viewcounts_key` staticmethod, nor (b) using it. Pick one.